doc/networks.md

```diff
@@ -267,16 +267,16 @@ Install the OVN tools and configure the OVN integration bridge on the local node
 
 ```
 apt install ovn-host ovn-central
-ovs-vsctl set open_vswitch . \
-  external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
-  external_ids:ovn-encap-type=geneve \
-  external_ids:ovn-encap-ip=n.n.n.n \ # The IP of your LXD host on the LAN
+ovs-vsctl set open\_vswitch . \
+  external\_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
```

Comment:
@stgraber these are in a code block and dont need escaping.

Reply:
They render `\_` which is incorrect.

Reply:
Ah, so it's vim getting confused I guess

Reply:
Dropped that commit